coursedata/adventures/en.yaml

```diff
@@ -904,6 +924,17 @@ adventures:
                     print choices at random
                     ```
                 start_code: "print Welcome to your own rock scissors paper!"
+            2:
+                story_text: |
+                    ## Rock, paper, scissors
+                    In this level you can practise using the variables, so that you can make the rock, paper, scissors game in the next level!
+
+                    ## Example Hedy code
+                    ```
+                    choice is (fill in what you choose)
```

Comment:
Zou het niet beter zijn om hier een _ te gebruiken ipv (fill in what you choose)?